joomla/joomla-cms
# [com_finder] maps view: Indexed content counting and other improvements

Pull Request for Improvement.
#### Summary of Changes

This PR adds some improvements to Smart Search Content Maps:
1. Add a label to each content map title (it was missing)
2. Adds link to view only the first level sublevels in the blue badges.
3. Adds counting and corresponding links to "Indexed Content" view.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/15053111/de178730-12f8-11e6-89ba-126498614155.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/15053093/cc6847cc-12f8-11e6-8aab-67c3bb3fdaa3.png)
#### Testing Instructions

3 simple test needed.
1. Apply patch
2. Go to Smart Search -> Indexed Content and run the index
3. Go to Smart Search -> Content Maps
4. Click on a content map "Title" and check that in that row the select box is checked.
   
   ok = test 1 passed
5. Click on the blue level 1 badge. and confirm it showns only the sublevels after click. Clear the filter after.
   
   ok = test 2 passed
6. Confirm the published/unpublished counting is working properly and they link to the index content view with the corresponding filter.
   Please note that the content_map will only be activated if PR https://github.com/joomla/joomla-cms/pull/10256 is also installed, but you can check the resulting URL (check the filter is there) for this test propose.
   
   ok = test 3 passed
